src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Confusing variable name, use noun for variables.
```suggestion
    protected String userName;
```

Reply:
Thanks for your suggestions. I have improved the naming for a few variables to avoid confusion.